main.py

```diff
@@ -1,10 +1,43 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_name, player_data in data.items():
+        self_race, created = Race.objects.get_or_create(
```

Comment:
Remove `self_` prefix from the `self_race` variable, it does not make the code more descriptive.